### PR TITLE
Extend and Validate Configuration Name length

### PR DESCRIPTION
**Pull request by shamoh19**
Fix for below improvement/bug

https://github.com/pimcore/data-importer/issues/99

### CODE REVIEW

File: src/Migrations/Version20211110093612.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_document` MODIFY `configuration` VARCHAR(80);');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // not needed
```
Comment (blankse) [inline src/Migrations/Version20211110093612.php:38]:
```suggestion
        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_document` MODIFY `configuration` VARCHAR(50);');
```

File: src/Migrations/Version20211110093612.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version20190614135736 extends AbstractPimcoreMigration
```
Comment (dvesh3) [inline src/Migrations/Version20211110093612.php]:
```suggestion
class Version20211110093612 extends AbstractPimcoreMigration
```

File: src/Migrations/Version20211110093612.php (hunk 1 above)
Comment (shamoh19) [reply src/Migrations/Version20211110093612.php:38]: @blankse Any reason for reverting back to 50 again? In Front end validation and while creating, its 80 only

Comment (blankse) [reply src/Migrations/Version20211110093612.php:38]: No I see no reason.

File: src/Migrations/Version20211110093612.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version20211110093612 extends AbstractPimcoreMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_document` MODIFY `configuration` VARCHAR(80);');
```
Comment (dvesh3) [inline src/Migrations/Version20211110093612.php:28]: please add migrations for `plugin_datahub_workspaces_asset` & `plugin_datahub_workspaces_object` tables as well

File: src/Migrations/Version20211110093612.php (hunk 1 above)
Comment (shamoh19) [reply src/Migrations/Version20211110093612.php:38]: @blankse Updated to 80 as we are using that value in Front end validation and while creating

**Review by shamoh19 — COMMENTED** [review]
Appropriate validation message in case of length exceeds 80

**Review by dvesh3 — COMMENTED** [review]
Also, adapt changes in Installer for new installations https://github.com/pimcore/data-hub/blob/d7bc9849ff8064ae718c034861f68d63683a4613/src/Installer.php#L48

File: src/Migrations/Version20211110093612.php (hunk 3 above)
Comment (shamoh19) [reply src/Migrations/Version20211110093612.php:28]: Done

File: src/Migrations/Version20211110093612.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // not needed
+        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_document` MODIFY `configuration` VARCHAR(80);');
+        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_asset` MODIFY `configuration` VARCHAR(80);');
+        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_object` MODIFY `configuration` VARCHAR(80);');
```
Comment (dvesh3) [inline src/Migrations/Version20211110093612.php]:
```suggestion
        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_document` MODIFY `configuration` VARCHAR(50);');
        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_asset` MODIFY `configuration` VARCHAR(50);');
        $this->addSql('ALTER TABLE `plugin_datahub_workspaces_object` MODIFY `configuration` VARCHAR(50);');
```
I would downgrade it to 50 as the behavior should be same for lower versions.

**Comment by dvesh3** [comment]
@shamoh19 thanks a lot!

**Pull request merged**